markdown/ch1.markdown

```diff
@@ -305,7 +313,7 @@ y = np.reshape(y, (1, len(y)))
 print(y)
 ```
 
-In order to do broadcasting, the two arrays have to have the same number of dimensions and the sizes of the dimensions need to match (or be equal to 1).
+In order to do broadcasting, the two arrays have to have the same number of dimensions and the sizes of the dimensions need to match or be equal to 1.
```

Comment:
I'm well aware of that. I was just arguing that this is a bit of syntactic niceness on top of the "basic" broadcasting rules, if that makes sense. I don't want to copy over the whole numpy broadcasting page in place of this sentence.